sixtyfps_compiler/passes/remove_aliases.rs

```diff
@@ -184,7 +184,7 @@ pub fn remove_aliases(component: &Rc<Component>, diag: &mut BuildDiagnostics) {
                         .or_default()
                         .is_set = true;
                 } else {
-                    drop(d);
+                    drop(std::mem::take(d));
```

Comment:
This is wrong.
If this compile the drop of d can be removed. (but not the drop of elem)
mem::take is wrong

Reply:
I dropped this drop.